src/index.html

```diff
@@ -7,8 +7,62 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="./reset.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a class="header__link" href="#home">
+        <img
+          class="header__logo"
+          src="./images/logo.png"
+          alt="Logo"
+        >
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#" class="nav__link is-active">
+              apple
+            </a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">
+              samsung
+            </a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">
+              smartphones
+            </a>
+          </li>
+          <li class="nav__item" data-qa="hover">
+            <a href="#" class="nav__link">
```

Comment:
please add `data-qa` for `a` element
![image](https://github.com/mate-academy/layout_moyo-header/assets/22824431/70a6a444-747c-4f88-841a-b9684663bf02)